app/assets/javascripts/auth_source_ldap_test_connection.js

```diff
@@ -0,0 +1,26 @@
+function testConnection(test_connection_url){
```

Comment:
I don't remember why exactly I had added a separate file, but I remember having a lot of trouble including the client side javascript because of the two-pane layout. But now, I tried testing it out by moving the function into `auth_source_ldap.js` and everything was working fine. so i guess its okay .